build/pre-integration-test.sh

```diff
@@ -52,6 +52,10 @@ echo "Copying test artifacts to install location..."
 rm -rf ${SIT}/wlp/usr/servers/fhir-server/userlib/*
 cp -pr ${WORKSPACE}/operation/fhir-operation-test/target/fhir-operation-*-tests.jar ${SIT}/wlp/usr/servers/fhir-server/userlib/
 
+echo "Copying profiles into the server"
+cp -pr ${WORKSPACE}/conformance/fhir-ig-davinci-pdex-formulary/target/fhir-ig-davinci-pdex-formulary-*.jar ${SIT}/wlp/usr/servers/fhir-server/userlib/
+cp -pr ${WORKSPACE}/conformance/fhir-ig-carin-bb/target/fhir-ig-carin-bb-*.jar ${SIT}/wlp/usr/servers/fhir-server/userlib/
+cp -pr ${WORKSPACE}/conformance/fhir-ig-us-core/target/fhir-ig-us-core-*.jar ${SIT}/wlp/usr/servers/fhir-server/userlib/
```

Comment:
I was thinking our zip installer copies these here already.  Maybe we just need to make sure `fhir-ig-davinci-pdex-formulary-*.jar` is included in that instead of adding this step to the pre-integration-test script?

Reply:
that's a good question... I wasn't sure, and may have added this without really knowing the root cause of the error I was chasing.

Reply:
I removed that copy (letting the CI test it)